engine/src/main/java/org/terasology/config/BindsConfig.java

```diff
@@ -82,7 +82,7 @@ public BindsConfig() {
      * @return True if newInput has been bound. False otherwise.
      */
     public boolean isBound(Input newInput) {
-        return data.containsValue(newInput);
+        return data.containsValue(newInput) && newInput != null;
```

Comment:
This is a minor nitpick, but a "newInput != null; && data.containsValue(newInput)" will be much more efficient when newInput is null, because of the most programming languages handle AND (using short circuit logic).